events/voiceStateUpdate.js

```diff
@@ -152,9 +153,8 @@ module.exports = {
 		// Quaver was playing something - set pauseTimeout
 		await player.pause();
 		logger.info({ message: `[G ${player.guildId}] Setting pause timeout`, label: 'Quaver' });
-		if (player.pauseTimeout) {
-			clearTimeout(player.pauseTimeout);
-		}
+		// When setting a pauseTimeout, clear pauseTimeout at any cost as failsafe
+		clearTimeout(player.pauseTimeout);
```

Comment:
`player.pauseTimeout` is always `undefined` here as well. This is a redundant statement and will for now stay here as a failsafe.